pkgs/development/haskell-modules/configuration-common.nix

```
@@ -612,6 +612,24 @@ self: super: {
     '';
   });
 
+  d-bus = let
+    # The latest stack release is missing necessary patches for recent compilers
```

```suggestion
    # The latest release on hackage is missing necessary patches for recent compilers
```
This lib is not on stackage.